docs/reference/mapping/params/doc-values.asciidoc

```diff
@@ -17,6 +17,12 @@ makes this data access pattern possible. They store the same values as the
 sorting and aggregations. Doc values are supported on almost all field types,
 with the __notable exception of `text` and `annotated_text` fields__.
 
+Numeric types, such as `long` and `double`, can also be queried when they are
```

Comment:
may I ask why you mention only long and double?

Reply:
Wasn't meant to be an exhaustive list. I've copied this phrasing from a different part of the docs. I've added a link on "Numeric types" now so that someone interested can look up all the types this applies to.

Reply:
thanks!